gcp_variant_transforms/libs/vcf_field_conflict_resolver_test.py

```diff
@@ -28,6 +28,9 @@ def setUp(self):
     self._resolver = vcf_field_conflict_resolver.FieldConflictResolver()
     self._resolver_allele = vcf_field_conflict_resolver.FieldConflictResolver(
         split_alternate_allele_info_fields=True)
+    self._resolver_all = vcf_field_conflict_resolver.FieldConflictResolver(
+        resolve_all=True
```

Comment:
nit: please put closing bracket on the same line

Reply:
Right! Thanks.